src/parcel.js

```diff
@@ -103,7 +103,7 @@ export default class Parcel extends React.Component {
 
       return React.createElement(
         this.props.wrapWith,
-        { ref: this.handleRef, style: this.props.wrapStyle },
+        { ref: this.handleRef, style: this.props.wrapStyle, className: this.props.className },
```

Comment:
Perhaps it would be better to name this as `props.wrapClassName`, to match the naming of the `wrapWith` and `wrapStyle` props?

Also, could you add a test for this change?

Reply:
Absolutely, that makes loads sense. I'll add some tests.